packages/yourloops/pages/patient/caregivers/table.tsx

```diff
@@ -30,7 +30,6 @@ import React from "react";
 import { useTranslation } from "react-i18next";
```

Comment:
Maybe changing this file was a little out of scope ?
There is enough changed already I think, it can be done in another PR.